apps/site/assets/ts/schedule/components/__schedule.d.ts

```diff
@@ -102,7 +102,7 @@ export interface ScheduleNote {
   peak_service: string;
   offpeak_service: string;
   exceptions: ServiceException[];
-  alternate_text?: string;
+  alternate_text: string | null;
```

Comment:
@meagonqz correct me if I'm wrong, but I thought that `varName?: dataType` was exactly identical to `varName: dataType | null`. Or did Pronto or something complain?